docs/index.asciidoc

```diff
@@ -58,7 +58,7 @@ filter plugins.
 
 Convert a field's value to a different type, like turning a string to an
 integer. If the field value is an array, all members will be converted.
-If the field is a hash, no action will be taken.
+If the field is a hash, nil, or an empty string, no action will be taken.
```

Comment:
comment refers to functionality that was rolled back

Reply:
doh! ... change incoming.